test/performance/1-basic-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+REPEAT=$1
+
+echo 'azmi - build executable'
+dotnet build src/azmi-commandline/azmi-commandline.csproj
```

Comment:
dotnet publish ./src/azmi-commandline/azmi-commandline.csproj --configuration Release --self-contained true /p:PublishSingleFile=true --runtime linux-x64
Is what we deliver to users.

Reply:
Fixed in all three scripts.